<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneOptionsWdgt</class>
 <widget class="QWidget" name="CloneOptionsWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Repository Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="txtCloneMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bare</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mirror</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3">
    <widget class="QGroupBox" name="grpSubmodules">
     <property name="title">
      <string>Submodules</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Depth:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="chkInitSubmodules">
        <property name="text">
         <string>Initialize submodules</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkSubmodulesRecursive">
        <property name="text">
         <string>Recursive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="txtSMCloneDepth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="specialValueText">
         <string>(full depth)</string>
        </property>
        <property name="prefix">
         <string>Depth: </string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Branch:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="txtBranch">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>(default branch)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="chkSingleBranch">
     <property name="text">
      <string>Single Branch</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Depth:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="txtCloneDepth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="specialValueText">
      <string>(full depth)</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
